mulighetsrommet-arena-adapter/src/test/kotlin/no/nav/mulighetsrommet/arena/adapter/events/processors/TiltakdeltakerEventProcessorTest.kt

```diff
@@ -163,21 +169,21 @@ class TiltakdeltakerEventProcessorTest : FunSpec({
 
             val event = createArenaTiltakdeltakerEvent(Insert) { it.copy(REG_DATO = regDatoBeforeAktivitetsplanen) }
 
-            consumer.handleEvent(event).shouldBeLeft().should { it.status shouldBe Ignored }
+            consumer.handleEvent(event).shouldBeRight().should { it.status shouldBe ArenaEntityMapping.Status.Ignored }
         }
 
         test("should be ignored when dependent tiltaksgjennomforing is ignored") {
             val events = ArenaEventRepository(database.db)
-            events.upsert(
-                createArenaTiltakgjennomforingEvent(Insert, status = Ignored) {
-                    it.copy(TILTAKGJENNOMFORING_ID = tiltaksgjennomforing.tiltaksgjennomforingId)
-                }
-            )
+            val entities = ArenaEntityMappingRepository(database.db)
+            val gjennomforingEvent = createArenaTiltakgjennomforingEvent(Insert, status = Processed) {
```

Comment:
Kanskje litt pirkete, men her tror jeg det holder å upserte entitymappingen (og droppe upsert av eventet) siden eventet ikke lengre er relevant for å finne ut om gjennomføringen er ignorert